hbase-server/src/test/java/org/apache/hadoop/hbase/regionserver/TestHRegionFileSystem.java

```diff
@@ -184,6 +185,62 @@ public void testBlockStoragePolicy() throws Exception {
       TEST_UTIL.shutdownMiniCluster();
     }
   }
+  @Test
+  public void testMobStoreStoragePolicy() throws Exception {
+    TEST_UTIL = new HBaseTestingUtil();
+    Configuration conf = TEST_UTIL.getConfiguration();
+    TEST_UTIL.startMiniCluster();
+    Table table = TEST_UTIL.createTable(TABLE_NAME, FAMILIES);
+    assertEquals("Should start with empty table", 0, TEST_UTIL.countRows(table));
+    HRegionFileSystem regionFs = getHRegionFS(TEST_UTIL.getConnection(), table, conf);
+    try (Admin admin = TEST_UTIL.getConnection().getAdmin()) {
+      ColumnFamilyDescriptorBuilder cfdA = ColumnFamilyDescriptorBuilder.newBuilder(FAMILIES[0]);
+      cfdA.setValue(HStore.BLOCK_STORAGE_POLICY_KEY, "ONE_SSD");
```

Comment:
Should we add a new key: "hbase.hmobstore.block.storage.policy"
 
I am not sure if this JIRA is a bug or a new feature. But teh change will definitely change behaviour for exisdting users once this patch is merged

Reply:
ok, i agree. what about the "hbase.hmobstore.block.storage.policy" default value? equals hstore  or just none?

Reply:
I don't think we need to add this new key, because when we creat a table, we set the `STORAGE-POLICY` configuration or ues the default policy (`HStore.BLOCK_STORAGE_POLICY_KEY`), which means that all data under this column family should follow this policy.
So, the storage strategy for MOB should be consistent with the column family.

Reply:
We discovered this issue when we found incorrect storage capacity in our cluster. We believe this is a bug, so I fix it like this pr, but I'm not sure how this is determined in our community. @NihalJain @guluo2016

Reply:
> We believe this is a bug, so I fix it like this pr

I agree with you.

> but I'm not sure how this is determined in our community. 

In my personal opinion, the storage strategy for MOB should be consistent with the column family.
Perhaps we can discuss it with Nihal and others together.

Reply:
I am fine with consolidating with mobstore as that is the right way. My only concern is we may unknowingly start moving data around if this silently lands with a new version upgrade. May be we are okay to just add to release note, since that would be a behavioral change (due to bug).

Hey @Apache9 WDYT?

Reply:
This PR has been around for a while, may not have noticed, remind. @Apache9